Maple2.Server.Game/GameServer.cs

```diff
@@ -25,6 +25,7 @@ public class GameServer : Server<GameSession> {
     private readonly ConcurrentDictionary<int, PremiumMarketItem> premiumMarketCache;
     private Dictionary<int, Shop> shopCache;
     private Dictionary<int, Dictionary<int, ShopItem>> shopItemCache;
+    public AiManager aiManager { get; init; }
```

Comment:
public properties should be capitalized, also why is this needed?

Reply:
forgot to remove lol